utils/podman/Makefile

```diff
@@ -93,10 +93,12 @@ endef
 define Package/podman/install
 	$(call GoPackage/Package/Install/Bin,$(1))
 	$(INSTALL_DIR) $(1)/etc/containers
+	chmod 0755 $(1)/etc/containers
```

Comment:
This is unneeded, as `$(INSTALL_DIR)` already implies `0755`, see
https://github.com/openwrt/openwrt/blob/master/rules.mk#L284

Reply:
Fixed. Thank you.